templates/community/docs/search-results.html

```diff
@@ -0,0 +1,12 @@
+{% with
+    title="Ubuntu Community Docs",
+    query=query,
+    results=results,
+    search_action="/community/docs/search",
```

Comment:
```suggestion
    search_action="/community/search",
```
Currently it leads to a 404

Reply:
It also causes the `Next` link at the bottom of the page to fail